### ISSUE TITLE
Test for WCAG technique G195 (visible focus indicator)

### ISSUE THREAD

**Issue by kevee** [quailjs/quail#78]
Create a test for the following WCAG 2.0 technique: http://www.w3.org/TR/WCAG20-TECHS/G195.html

The objective of this technique is enhance the focus indicator in the browser, by creating a highly visible one in the content. The default focus indicator in many browsers is a thin,dotted, black line. It can be difficult to see the line when it is around a form element which already has an outline, when the focused element is inside a table cell, when the focused element is very small, or when the background of the page is a dark color.

In this technique, when the user places focus on an element, using the mouse, tab key, arrow keys, keyboard shortcuts, or any other method, the application makes that focus more visible, using a combination of a highly contrasting color, a thick line, and other visual indicators such as a glow.

Test Procedure
- Place focus on each focusable user interface element on the page using the mouse and keyboard.
- Check that there is a highly visible focus indicator.

related to WCAG C15 (using css for focus)
